tests/ttnn/integration_tests/vit/test_performance_ttnn_optim_sharded_vit.py

```diff
@@ -61,21 +61,13 @@ def test_performance_vit_encoder(device, use_program_cache, model_name, batch_si
         device=device,
     )
 
-    hidden_states = ttnn.from_torch(
-        torch_hidden_states,
-        dtype=ttnn.bfloat8_b,
-        layout=ttnn.TILE_LAYOUT,
-        device=device,
-        memory_config=ttnn.L1_MEMORY_CONFIG,
-    )
+    hidden_states = ttnn.from_torch(torch_hidden_states, dtype=ttnn.bfloat8_b, layout=ttnn.TILE_LAYOUT)
```

Comment:
is removal of memory_config intentional?
It was `L1_MEMORY_CONFIG`, but the default value in **from_torch** is `memory_config: Optional[ttnn.MemoryConfig] = None,`

Reply:
Yes. There was a bug in this test as the inputs were deallocated after each iteration. 

We now move the tensor to device in the inference loop and we use the memory config there.